api/wd/index.md

```diff
@@ -571,25 +593,25 @@ browser.evaluateAsync(3, 6, (a, b) => {
 });
 ```
 
-### Cookies
+## Cookies
 
 Cookie objects have the following general structure:
 
-```js
+```json
 {
-  "name",
-  "value",
-  "path" = "/",
-  "domain" = currentPageDomain,
-  "expiry" = endOfCurrentSession,
-  "secure" = false,
+  "name":  "<name>",
+  "value":  "<value>",
+  "path": "/",
+  "domain": "<currentPageDomain>",
+  "expiry": "<endOfCurrentSession>",
+  "secure": false
```

Comment:
I think the old syntax was more suggestive of those values being defaults, since that's the JS destructure syntax for defaults

Reply:
hmm I'm not  sure. I reverted it to before